app/graphql/resolvers/freak_update_resolver.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Resolvers
+  class FreakUpdateResolver < Resolvers::Base
+    argument :id, GraphQL::Types::ID, required: true
+    argument :first_name, String, required: true
+    argument :last_name, String, required: true
+    argument :description, String, required: true
+    argument :email, String, required: true
+    argument :norm_id, GraphQL::Types::ID, required: true
+    argument :role_id, GraphQL::Types::ID, required: true
+    argument :level_id, GraphQL::Types::ID, required: true
```

Comment:
How do you expect the client's to send you the data: will they send all the attributes, and some of them will be different from the initial ones, or will they send you just the attributes they want to change instead of all?

Reply:
You are right. For test was ok because factory bot will create a freak, then we can change what we want for that freak. If we don't want to change a parameter we can add old value with (freak.value: for example: freak.first_name). I will resolve this issue in next PR. Thanks.

Reply:
There's nothing to be right here, I wasn't trying to prove a point.

It's really just a question to see how you thought this out since both approaches are valid. :man_shrugging: 

What I'd do is to speak with some of the client-side devs & see which approach would help them more.

Reply:
I resolved that issue approach. Now we need just id, the rest of the parameters are optional. Thanks